### PR TITLE
Branch lk 2

### PR DESCRIPTION
**Pull request by LilianaKor**
Small changes in table 1 and I also want to add my lesson-8.feature file

### CODE REVIEW

File: lesson_8.feature [hunk 1]
```diff
@@ -19,7 +19,7 @@ Feature: Lesson 8
     Then Wait 2 seconds
 
 
-  Scenario: Verify privacy Manager with table
+  Scenario: Verify privacy 1 Manager with table 1
```
Comment (andrey-baykov) [inline lesson_8.feature:22]: Please rename scenario. Make it more clear.

File: lesson_8.feature [hunk 2]
```diff
@@ -41,9 +41,9 @@ Feature: Lesson 8
     Examples:
       | email                     | password   | result |
       | pcs.class1223@gmail.com   | Qwertyui1@ | true   |
-#      | pcs.class1223+2@gmail.com | Qwertyui2@ | true   |
-#      | pcs.class1223+3@gmail.com | Qwertyui3@ | true   |
-#      | pcs.class1223+4@gmail.com | Qwertyui4@ | false  |
+      | pcs.class1223+2@gmail.com | Qwertyui2@ | true   |
+      | pcs.class1223+3@gmail.com | Qwertyui3@ | true   |
```
Comment (andrey-baykov) [inline lesson_8.feature:45]: Don't use it. Password will be changed tomorrow.

Comment (andrey-baykov) [reply lesson_8.feature:45]: Delete this row.

**Review by andrey-baykov — CHANGES_REQUESTED** [review]

File: lesson_8.feature (hunk 1 above)
Comment (LilianaKor) [reply lesson_8.feature]: Done, You're right it didn't make sense

File: lesson_8.feature (hunk 2 above)
Comment (LilianaKor) [reply lesson_8.feature]: ok. I got it. Done

**Pull request merged**